upload/MultiFile.php

```diff
@@ -1,77 +1,83 @@
 <?php
+
 namespace upload;
 
 class MultiFile
 {
-  public $Files = array();
-  public $Errors = array();
+    public $Files = [];
+    public $Errors = [];
 
-  function __construct($key)
-  {
-    for ($i=0; $i < count($_FILES[$key]['name']); $i++) {
-      if (is_uploaded_file($_FILES[$key]['tmp_name'][$i])) {
-        $data = array(
-          'name' => $_FILES[$key]['name'][$i],
-          'tmp_name' => $_FILES[$key]['tmp_name'][$i]
-          );
-        $this->Files[] = new File($key, $data);
-      }
+    public function __construct($key)
```

Comment:
__[phpmd]__
* __construct accesses the super-global variable $_FILES.

<!-- Comment by [SideCI](https://sideci.com) -->